### ISSUE TITLE
Cannot export png on macOS because of wrong libpng version

### ISSUE THREAD

**Issue by jacogasp** [LibreSprite/LibreSprite#404]
##### LibreSprite version

```
1.0.0
```

##### Describe the bug

An error occurs when exporting sprite sheet because of incompatible version of libpng. The png file is created but it is empty/corrupted.

##### To Reproduce

Export a sprite with `File-> Export Sprite Sheet` then click `Export` and the following error occour:

```
libpng: Application build with libpng-1.4.12 but running with 1.6.39
```

##### Expected behaviour
Export png to disk without errors.

##### OS/Hardware
- Operative System: macOS 13.3.1
- CPU: M2

##### Screenshots  
<img width="1792" alt="Screenshot 2023-05-30 at 18 55 10" src="https://github.com/LibreSprite/LibreSprite/assets/6513727/19131360-fb5c-4f7a-85c0-a4e711e38091">

<img width="1792" alt="Screenshot 2023-05-30 at 18 55 21" src="https://github.com/LibreSprite/LibreSprite/assets/6513727/e6738b52-94bb-407a-a081-40ffa99ca474">

**Comment by jjhaggar** [comment]
Hi there, I think there is already a solution for that.

Quoted from **Linux-Gamer** in the Discord server:

> You need to install the version of libpng that it says. So 1.4.12 - potentially from here, dunno how to install it though: https://sourceforge.net/projects/libpng/files/libpng14/older-releases/1.4.12/

That should fix the issue (hopefuly), please try it and let us know if it worked :)

**Comment by Geolykt** [comment]
According to <https://github.com/LibreSprite/LibreSprite/issues/403#issuecomment-1562597099> the bug was already fixed, but I cannot verify that myself

**Comment by jacogasp** [comment]
> According to [#403 (comment)](https://github.com/LibreSprite/LibreSprite/issues/403#issuecomment-1562597099) the bug was already fixed, but I cannot verify that myself

Hi @Geolykt and @jjhaggar,
thanks for the quick response.

Indeed, I tried to download the "new" version from https://github.com/LibreSprite/LibreSprite/issues/403#issuecomment-1562597099 and it works perfectly.

I did not notice that a new version came out, since it is still labelled v1.0.0 and I downloaded Libresprite, for the first time, only a couple of weeks ago.
